template/package.json

```diff
@@ -4,7 +4,7 @@
   "scripts": {
     "start": "parcel src/index.html --open",
     "build": "parcel build src/index.html",
-    "test": "jest --watch"
+    "test": "jest src --watch"
```

Comment:
pointing to src now since that's where tests should exist (this was causing issues with custom blueprints locally in projects)